Assets/SEE/Controls/Actions/ShowEdges.cs

```diff
@@ -29,6 +34,23 @@ public class ShowEdges : InteractableObjectAction
         /// </summary>
         private AbstractSEECity codeCity;
 
+        /// <summary>
+        /// A token that's used to cancel the transitive edge toggling.
+        /// We need to use this instead of the edge operator's built-in conflict mechanism
+        /// because the edge operator only controls a single edge, instead of the whole transitive closure.
+        /// </summary>
+        private CancellationTokenSource edgeToggleToken;
+
+        /// <summary>
+        /// The toggle that is used to determine whether an edge is currently selected.
+        /// </summary>
+        private const string EdgeIsSelectedToggle = "IsSelected";
```

Comment:
All private fields must start with a lower letter, no matter whether they are constant or not.
It's the standard Axivion rule.

Reply:
Yes, fixed it.